### ISSUE TITLE
"exec format error" on Raspberry Pi 4

### ISSUE THREAD

**Issue by LeppyThe1st** [tiredofit/docker-openldap-fusiondirectory#13]
Hi,

I am attempting to run this, OpenLDAP and FusionDirectory on a raspberry pi 4 however on run the only log I get is 
`standard_init_linux.go:211: exec user process caused "exec format error",`

I believe that this is caused by these containers not being built for Arm64 devices however I am not 100% certain of this. Any help on this would be amazing.
Thanks!

**Comment by tiredofit** [comment]
That's correct - They would need to be tweaked to run on Arm. I don't have a device to build or test on anymore, but it should be easy to do. You'd need to build all the base images (tiredofit/alpine) and swap out the s6 overlay system with the arm 64 version, and then build tiredofit/openldap, then tiredofit/openldap-fusiondirectory. On the web frontend side it's more of the same, but starting with tiredofit/alpine, then tiredofit/nginx, then tiredofit/nginx-php-fpm, then tiredofit/fusiondirectory.

**Comment by LeppyThe1st** [comment]
Alright. I'm a novice when it comes to Docker however, I'll attempt to get this to work. Thank you for the information!!
